x/mongo/driver/errors.go

```diff
@@ -230,8 +246,24 @@ func (e Error) Retryable() bool {
 			return true
 		}
 	}
-	if strings.Contains(e.Message, "not master") || strings.Contains(e.Message, "node is recovering") {
-		return true
+
+	return false
+}
+
+// RetryableWrite returns true if the error is retryable for a write operation
+func (e Error) RetryableWrite(wireVersion *description.VersionRange) bool {
```

Comment:
I can't comment on the `Retryable` method because it wasn't changed, but can we change its name to `RetryableRead`?

Reply:
done